### PR TITLE
Creation and design of checkboxs components

### PR DESCRIPTION
**Pull request by Ocosmee**
# Description
 - Creation and design of checkbox components from issue #39 
 
 # Testing
- Run npm install
- Run npx react-native start
- Run npx react-native run-android
 
  ## Screenshots 
Screen

### CODE REVIEW

**Review by JordiGN — DISMISSED** [review]
Nice! I'm glad you could solve the issues with the branches

File: tribomobile/App.js [hunk 1]
```diff
@@ -22,14 +22,26 @@ import RecomendationScreen from './screens/recomendationScreen';
...
 const App: () => React$Node = () => {
   return (
     <>
       <StatusBar barStyle="dark-content" />
-      <SafeAreaView>
-        <WelcomeGuestScreen />
+      <SafeAreaView style={{flex:1,justifyContent:'center'}}>
+        <Text style={{marginLeft:35, marginBottom:10}}>{titlesText.titleRegistrationVenta}</Text>
+        <RadioButtomCustom/>
+        <Text style={{marginLeft:35, marginBottom:10, marginTop:10}}>{titlesText.titleRegistrationFormasPago}</Text>
```
Comment (develiasdaniel) [inline tribomobile/App.js]: i would recommend put this into style sheet

**Review by develiasdaniel — DISMISSED** [review]
Nice job

**Review by Ing-Heber — DISMISSED** [review]
Nice work Cosme

File: tribomobile/App.js (hunk 1 above)
Comment (LeyaimMagma) [reply tribomobile/App.js]: x2

**Review by LeyaimMagma — CHANGES_REQUESTED** [review]

**Review by Ing-Heber — DISMISSED** [review]
Niceeeeeee!

**Review by Ing-Heber — DISMISSED** [review]

**Review by Ing-Heber — APPROVED** [review]
good job cosme!!

**Review by JordiGN — APPROVED** [review]
Cool design, it looks pretty much like the one in the design document! Great job!

**Review by LeyaimMagma — APPROVED** [review]

**Review by luisarmandodv — APPROVED** [review]

**Pull request merged**